internal/state/eventbus.go

```diff
@@ -18,11 +18,12 @@ const (
 	SigFailed
 	SigTimeout
 	DepositReceive
+	WithdrawRequest
```

Comment:
Please add  it after BlockScanned, enum index cannot be replace

Reply:
fixed.